lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -11,6 +11,23 @@ include_once 'includes/KudosController.php';
 include_once 'includes/KudosTransformer.php';
 
 
+function dosomething_kudos_menu() {
+  $items = array();
+
+  // Admin Kudos configuration page.
+  $items['admin/config/dosomething/dosomething_kudos'] = array(
```

Comment:
ahhh I asked yesterday if i should make a new form but I guess you missed it. ill move it to reportback